modules/fedimint-ln-client/src/lib.rs

```diff
@@ -418,7 +438,13 @@ async fn ln_operation(client: &Client, operation_id: OperationId) -> anyhow::Res
 }
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-enum LightningMeta {
+pub struct LightningMeta {
+    variant: LightningMetaVariants,
+    extra_meta: serde_json::Value,
```

Comment:
What are you thinking of using this for? Just curious

Reply:
When an outgoing payment reaches a terminal state, it would be nice to know which invoice was paid.